test/trigger_cellranger_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# note, this test must be run from the workflow root for the relative paths to work out.
```

Comment:
Same as below, this comment should mention this is only true for the default `TEST_DIR` since it's configurable now!